vue/components/ui/atoms/container/container.vue

```diff
@@ -180,7 +199,7 @@ export const Container = {
         }
     },
     methods: {
-        onButtonIconClick(event, buttonId) {
+        onHeaderButtonClick(event, buttonId) {

```

Comment:
Maybe emit `header-button:click:buttonId` too, so then you can consume it with `v-on:header-button:click:add`.